Source/WebCore/loader/FTPDirectoryParser.cpp

```diff
@@ -145,10 +145,7 @@ FTPEntryType parseOneFTPLine(std::span<LChar> line, ListState& state, ListResult
                             while (pos < linelen && isASCIIDigit(line[pos]))
                                 pos++;
                             if (pos < linelen && line[pos] == ',') {
-                                unsigned long long seconds = 0;
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-                                sscanf(byteCast<char>(p.subspan(1)).data(), "%llu", &seconds);
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
+                                uint64_t seconds = parseIntegerAllowingTrailingJunk<uint64_t>(spanConstCast<const LChar>(p.subspan(1))).value_or(0);
```

Comment:
So annoying that we have to cast just to _add_ `const`.

Reply:
I'll spend a few minutes seeing if we can avoid it somehow.

Reply:
I used `StringView { }` instead which I think is a bit nicer.